test/mbta_v3_api/facilities/repo_test.exs

```diff
@@ -1,17 +1,54 @@
 defmodule Facilities.RepoTest do
   @moduledoc false
   use ExUnit.Case
+  import MBTAV3API.Support.Factory
+  import Mock
 
   alias JsonApi.Item
+  alias MBTAV3API.Facilities
+  alias MBTAV3API.Facilities.Facility
   alias MBTAV3API.Facilities.Repo
 
+  @item build(:facility_data)
```

Comment:
This use of a factory's default values "as a fixture" seems a bit fragile. Conventionally, not providing an attribute value to a factory means you're fine with that attribute having _any_ valid value (even a random one) — this helps ensure tests don't accidentally depend on data you didn't intend them to. The approach in this test has a couple downsides comparatively, though they're minor:

1. Any time any of the default values for the factory are changed, even if conceptually the change is unrelated to this test, it will break and need to be updated.
2. The "input" and "output" for this test are split across the factory definition and the test file — it's not immediately clear where values like `349` in the "expected response" are coming from.

I do think this specific use of default values is pretty defensible, and if it's an existing pattern elsewhere in the test suite I don't think it needs to change here. Just food for thought 🥮